src/sidecar_comms/inbound.py

```diff
@@ -49,6 +49,12 @@ def handle_msg(data: dict, comm: Comm) -> None:
         msg = CommMessage(body=variables, handler="get_kernel_variables")
         comm.send(msg.dict())
 
+    if inbound_msg == "rename_kernel_variable":
```

Comment:
```suggestion
    elif inbound_msg == "rename_kernel_variable":
```

... and on subsequent ones. Will match at most one, right?

Reply:
Yeah, it can only be one. I'd like to eventually get away from this pattern and into better callback handling overall, not quite there yet though.